endpoint.go

```diff
@@ -890,6 +890,14 @@ func CreateOptionPortMapping(portBindings []types.PortBinding) EndpointOption {
 	}
 }
 
+// CreateOptionDNS function returns an option setter for dns entry option to
+// be passed to container Create method.
+func CreateOptionDNS(dns []string) EndpointOption {
+	return func(ep *endpoint) {
+		ep.generic[netlabel.DNSServers] = dns
+	}
+}
+
```

Comment:
We have added this to support the --dns server options. These servers will be passed on to HNS for configuring the container. In windows dns servers are specific to network adapters and are not universal as in linux